migrations/app/20171018122200_app_court_reports_add_workload_report_id.js

```diff
@@ -0,0 +1,13 @@
+exports.up = function (knex, Promise) {
+  var sql = 'ALTER TABLE app.court_reports ADD workload_report_id INT;'
+  return knex.schema
+          .raw('SET ARITHABORT ON')
+          .raw(sql)
+}
+
+exports.down = function (knex, Promise) {
+  var sql = 'ALTER TABLE app.court_reports DROP COLUMN workload_report_id;'
+  return knex.schema
+          .raw('SET ARITHABORT ON')
+          .raw(sql)
+}
```

Comment:
Think this file was also in my PR, we'll have to watch out that Git doesn't do something stupid